src/enkf/gridio_fv3reg.f90

```diff
@@ -441,13 +480,14 @@ subroutine readgriddata(nanal1,nanal2,vars3d,vars2d,n3d,n2d,levels,ndim,ntimes,f
     if (ps_ind > 0) then
       allocate(workprsi(nx_res,ny_res,nlevsp1))
       allocate(pswork(nx_res,ny_res))
-      fv3filename=trim(adjustl(filename))//"_dynvartracer"
-      call nc_check( nf90_open(trim(adjustl(fv3filename)),nf90_nowrite,file_id),&
-                      myname_,'open: '//trim(adjustl(fv3filename)) )
+!clt      fv3filename=trim(adjustl(filename))//"_dynvartracer"
+!clt      call nc_check( nf90_open(trim(adjustl(fv3filename)),nf90_nowrite,file_id),&
+!clt                      myname_,'open: '//trim(adjustl(fv3filename)) )
+      call fv3lamfilebank%get_idfn('delp',file_id,fv3filename)
       call read_fv3_restart_data3d('delp',fv3filename,file_id,workvar3d)  
        !print *,'min/max delp',ntile,minval(delp),maxval(delp)
-      call nc_check( nf90_close(file_id),&
-      myname_,'close '//trim(fv3filename) )
+!cltorg      call nc_check( nf90_close(file_id),&
+!clt      myname_,'close '//trim(fv3filename) )
```

Comment:
Better to remove clt

Reply:
All suggested changes are done as suggested! Thanks